rmw_zenoh_cpp/src/detail/event.cpp

```diff
@@ -192,26 +176,36 @@ void EventsManager::attach_event_condition(
       "RMW Zenoh is not correctly configured to handle rmw_zenoh_event_type_t [%d]. "
       "Report this bug.",
       event_id);
-    return;
+    return false;
```

Comment:
In line with suggestion to rename to `queue_is_empty`

```suggestion
    return true;
```